codespell_lib/_codespell.py

```diff
@@ -581,12 +581,15 @@ def parse_options(
     used_cfg_files = []
     for cfg_file in cfg_files:
         _cfg = configparser.ConfigParser()
-        _cfg.read(cfg_file)
+        try:
+            _cfg.read(cfg_file)
+        except configparser.Error:
+            continue  # ignore unparsable config file!
```

Comment:
I think it ought to tell you it's failed to read it (with the error) at least for some level of verbose logging. Is setup.cfg likely to be invalid generally, or only when someone has failed to write it properly.

Otherwise it's going to be hard to troubleshoot what's going on/why it's not working for you as expected.

Reply:
I had left out logging because this function did not log until now, just returning a status. But yes, some message or error status is needed.

Need to add tests too.

Reply:
It sort of inadvertently did via the stack trace! 😆